spec.md

```diff
@@ -161,9 +161,11 @@ Throughout this document, `<reference>` as a tag MUST be at most 128 characters
 
 The client SHOULD include an `Accept` header indicating which manifest content types it supports.
 In a successful response, the `Content-Type` header will indicate the type of the returned manifest.
+The registry SHOULD NOT include parameters on the `Content-Type` header.
+The client SHOULD ignore parameters on the `Content-Type` header.
```

Comment:
Language about whether clients can set parameters in Content-Type header when pushing?

Reply:
Oh, good call out! I've updated that section too. I really want to move all of this to a new top level section of the spec where we call out which RFCs we depend on and what conditions we have on those RFCs (like no parameters). But now doesn't feel like the right time for dramatic changes.